pkg/dpipe/dpipe_windows.go

```diff
@@ -23,6 +23,12 @@ type processInfo struct {
 	exe  string
 }
 
+func prepareCmd(cmd *dexec.Cmd) {
```

Comment:
Given the somewhat thorough peruse of child processes and how they are killed in the right order on Windows, I'm surprised this would be needed.